CodeGeneration/Sources/SyntaxSupport/KeywordSpec.swift

```diff
@@ -19,6 +19,7 @@ public struct KeywordSpec {
   /// The experimental feature the keyword is part of, or `nil` if this isn't
   /// for an experimental feature.
   public let experimentalFeature: ExperimentalFeature?
+  public let experimentalFeature2: ExperimentalFeature?
```

Comment:
I think it would be nicer to make this an array of experimental features + update the doc comment to say that the keyword is enabled if any of the experimental features are enabled.

Reply:
I know this stuff is performance-sensitive so I wasn't sure about introducing a refcounted field into the structure just for a one-off case where we happen to need two features for the same keyword. I'm also hoping I can just take this out again once the `.borrowingSwitch` feature is enabled, which will hopefully occur in the nearish term.

Reply:
This is just part of the code generation, so it’s not performance critical at all.